tests/modules/users/test_users_routes.py

```diff
@@ -16,11 +16,11 @@ def test_show_memberlist(client):
   response = client.get(flask.url_for('users.show_memberlist'))
   assert response.status_code == httplib.OK
 
-def test_view_profile(client):
+def test_view_member(client):
```

Comment:
big ups for actually considering this, i literally haven't ever thought about our tests lol